tests/the_forge_spec.rb

```diff
@@ -98,7 +98,7 @@
       .should match /<%\-\s+\|\s+.*?\$db,.*?\|\s+\-%>/m
     file("#{MODULE_PATH}pasture/templates/pasture_config.yaml.epp")
       .content
-      .should match /<%\-?\s+if\s+\$db\s+{\s+\-%>.*?:db:\s+<%=\s+\$db\s+%>.*?<%\-?\s+}\s+\-%>/m
+      .should match /<%\-?\s+if\s+\$db\s+!= (['"])none\1\s*{\s+\-%>.*?:db:\s+<%=\s+\$db\s+%>.*?<%\-?\s+}\s+\-%>/m
```

Comment:
Might want to replace the newly-inserted space character with `\s+` here? @kjhenner

Reply:
yep, good call.